test/functional/api/vim_spec.lua

```diff
@@ -5205,4 +5205,172 @@ describe('API', function()
     api.nvim__redraw({ win = 0, range = { 0, -1 } })
     n.assert_alive()
   end)
+
+  describe('nvim_cmd with plus #flags', function()
+    it('handles +flags correctly', function()
+      -- Write a file for testing +flags
+      command(':call writefile(["Line 1", "Line 2", "Line 3"], "testfile")')
+
+      -- Test + command (go to the last line)
+      local result = exec_lua([[
+        vim.cmd(vim.api.nvim_parse_cmd('edit + testfile', {}))
+        return vim.fn.line('.')
+      ]])
+      eq(3, result)
+
+      -- Test +{num} command (go to line number)
+      result = exec_lua([[
+        vim.cmd(vim.api.nvim_parse_cmd('edit +3 testfile', {}))
+        return vim.fn.line('.')
+      ]])
+      eq(3, result)
+
+      -- Test +/{pattern} command (go to line with pattern)
+      result = exec_lua([[
+        vim.cmd(vim.api.nvim_parse_cmd('edit +/Line\\ 2 testfile', {}))
+        return vim.fn.line('.')
+      ]])
+      eq(2, result)
+
+      -- Test +{command} command (execute a command after opening the file)
+      result = exec_lua([[
+        vim.cmd(vim.api.nvim_parse_cmd('edit +set nomodifiable testfile', {}))
+        return vim.bo.modifiable
+      ]])
+      eq(false, result)
+
+      -- Clean up
+      os.remove('testfile')
+    end)
+
+    it('handles ++ff correctly', function()
+      exec_lua [[
+        vim.cmd(vim.api.nvim_parse_cmd('edit ++ff=mac test.txt', {}))
+      ]]
+      eq('mac', api.nvim_get_option_value('fileformat', {}))
+      eq('test.txt', fn.fnamemodify(api.nvim_buf_get_name(0), ':t'))
+    end)
+
+    it('handles ++fileformat correctly', function()
```

Comment:
these tests can be combined.

please look for similar opportunities in the other tests. avoid verbose, single-purpose tests when multiple related things can be covered in a single test.